Lib/fontTools/ttLib/tables/otConverters.py

```diff
@@ -1569,6 +1569,19 @@ def xmlWrite(self, xmlWriter, font, value, name, attrs):
 	def xmlRead(self, attrs, content, font):
 		return safeEval(attrs["value"])
 
+class LookupFlag(UShort):
+	def xmlWrite(self, xmlWriter, font, value, name, attrs):
+		xmlWriter.simpletag(name, attrs + [("value", value)])
+		if value > 0:
+			flags = []
+			if value & 0x01: flags.append("rightToLeft")
+			if value & 0x02: flags.append("ignoreBaseGlyphs")
+			if value & 0x04: flags.append("ignoreLigatures")
+			if value & 0x08: flags.append("ignoreMarks")
+			if value & 0x10: flags.append("useMarkFilteringSet")
+			if value & 0xff00: flags.append("markAttachmentType")
```

Comment:
Would it make sense to add the value to the string? Say, `markAttachmentType[1]`